My-Spec-Tutorial.package/MyFirstWindow.class/instance/userTitle.st

```diff
@@ -1,10 +1,4 @@
-as yet unclassified
+private
 userTitle
-        "Find out user's title by checking the radio buttons."
         
-        radioMr state
```

Comment:
A switch statement is a code smell. It's often because you are trying to answer a question best answered by another object. In this case, the RadioButtonGroup already contains all the buttons, so it is best positioned to answer. Even though it is a part of the Kernel, it is as available to modification as your own code.